matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/AuthenticationService.kt

```diff
@@ -63,6 +64,13 @@ interface AuthenticationService {
      */
     fun getRegistrationWizard(): RegistrationWizard
 
+    /**
+     * Return a RefreshWizard, refresh our authentication with the homeserver.
+     *
+     * See [RefreshWizard] for more details.
+     */
+    fun getRefreshWizard(sessionId: String): RefreshWizard
```

Comment:
I do not think that we need to introduce a `RefreshWizard` in the API that the SDK exposes to the application.